Content.Server/Atmos/Monitor/Systems/AtmosMonitoringSystem.cs

```diff
@@ -215,9 +224,27 @@ private void OnAtmosUpdate(EntityUid uid, AtmosMonitorComponent component, ref A
             && component.GasThresholds == null)
             return;
 
+        // If monitoring a pipe network, get its most recent gas mixture
+        if (component.MonitorsPipeNet)
+            component.TileGas = GetGasPipeAirMixture(uid);
+
         UpdateState(uid, component.TileGas, component);
     }
 
+    private GasMixture? GetGasPipeAirMixture(EntityUid uid)
+    {
+        if (!TryComp<NodeContainerComponent>(uid, out var nodeContainer))
+            return null;
+
+        foreach (var node in nodeContainer.Nodes.Values)
+        {
+            if (node is PipeNode { } pipeNode)
+                return pipeNode.Air;
+        }
```

Comment:
Probably better to just use the node id and have it set in the component so the behavior isn't yml order-dependent. 

Like how we have to reference solutions by id/name.

Reply:
`nodeContainer.Nodes[component.NodeName]` is a common thing for atmos devices

Reply:
I can do that